dwitter/templates/base.html

```diff
@@ -37,23 +37,25 @@
       // Check for compatibility
       if (window.localStorage){
         var switchTheme = function(){
-          // Change the imported stylesheet from main.css to main_dark.css or vice versa
-          // Change localstorage theme to 'dark'
+          // Shift themes
           var theme = localStorage.getItem('theme');
-          if (theme == 'dark') {
-            localStorage.setItem('theme','');
+          if (theme && theme !== 'default') { 
+            localStorage.setItem('theme','black');
+          } else if (theme === 'black') {
+            localStorage.setItem('theme','green');
           } else {
-            localStorage.setItem('theme','dark');
+            localStorage.setItem('theme','default');
           }
           location.reload();
           // Call this function in the li to change the mode
         }
 
         // Do the same thing outside the function without changing localStorage to set correct theme
         var theme = localStorage.getItem('theme');
-        if (theme == 'dark'){
+        if (theme && theme !== 'default') {
           var l = document.createElement('link');
-          l.setAttribute('href', '{% static "theme_dark.css" %}');
+          l.setAttribute('href', '{% static "theme_'+theme+'.css" %}');
```

Comment:
Unfortunately this doesn't work.
The {%  %} template functions are run on the server and returns a string that gets included in the javascript. So, it doesn't have access to any javascript variables.

I think you'll have to either go back to redundant if statements, or make a dictionary to get the path values:

themePaths = {''dark": '{% static "theme_dark.css" %}',
                          "green": '{% static "theme_green.css" %}'}
and then 
l.setAttribute('href', themePaths[theme]);

Or something similar.

NOTE: not tested

PS: You should set up dwitter locally so you can test functionality

Reply:
I tried to host it but no success... I was expecting that will not work. I will edit it.